routes/error-tracker.js

```diff
@@ -193,19 +236,21 @@ function getHandler(req, res, next) {
         + url.parse(req.url, true).query['v'], err);
     }
   });
-
   if (params.debug === '1') {
     res.set('Content-Type', 'application/json; charset=ISO-8859-1');
-    res.status(statusCodes.OK).send(
-      JSON.stringify({
-        message: 'OK\n',
-        event: event,
-        throttleRate: throttleRate,
-      })).end();
+    res.status(statusCodes.OK);
+    res.send(
+        JSON.stringify({
+          message: 'OK\n',
+          event: event,
+          throttleRate: throttleRate,
+        })
+    ).end();
   } else {
     res.sendStatus(statusCodes.NO_CONTENT).end();
   }
   next();
 }
 
 module.exports = getHandler;
+getHandler.stackTraceConversion = stackTraceConversion;
```

Comment:
This is weird. Just export an object with both functions: `module.exports = {getHandler, stackTraceConversion};`